alchemist/alchemist-swingui/src/main/kotlin/it/unibo/alchemist/input/Keybinds.kt

```diff
@@ -32,7 +32,7 @@ enum class ActionFromKey {
 class Keybinds {
     companion object {
         private val filesystemPath = "${System.getProperty("user.home")}${File.separator}Alchemist${File.separator}"
-        private const val classpathPath = "it/unibo/alchemist/gui/"
+        private val classpathPath = "it${File.pathSeparator}unibo${File.pathSeparator}alchemist${File.pathSeparator}gui${File.pathSeparator}"
```

Comment:
Does it access a file, or does it load from classpath? In the latter case, `/` is to be used. The classpath is always separated by `/`

Reply:
That version of Keybinds.kt is outdated, in a later commit I fixed this issue.